content/cer.ttl

```diff
@@ -2,21 +2,31 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :cer
     skos:prefLabel "CER"@en;
+    skos:broader "feed";
+    skos:related :forex;
     skos:definition '''
       The feed type representing Currency Exchange Rate feeds.
-      This includes the feeds ADA-USD and FACT-ADA.
+      This includes feeds such as ADA-USD and FACT-ADA.
 
       ### Naming convention
 
-      For a feed of CER type, the feed name naming convention is `{{cur-a}}-{{curr-b}}`,
-      where the ordering follows the forex convention of 'base currency second' (see [base-quote](#baseQuote)).
-      Currency names of fiat follow ISO 4217.
-      For crypto base currencies, we defer to how it appears on CEXs.
-      For cardano native tokens, we defer to how it appears on the main DEXs.
-      For example: for the case of crypto-fiat `ADA-USD`; crypto-stable `ADA-iUSD`; crypto-base and other native asset `FACT-ADA`.
+      For a feed of CER type, the naming convention is
+      `{{base-currency}}-{{quote-currency}}` and ordering
+      follows forex convention (see [base-quote](#baseQuote)).
 
-      Beyond this, there is not yet precedence.
-      Currency names should stripped of `/` and any non-standard characters.
-      Ordering will be fleshed out as it arises.
-    ''' ;
-    skos:related :forex .
+      Fiat currency names follow ISO 4217.
+
+      Currency names are stripped of `/` and any non-standard
+      characters.
+
+      For crypto base currencies, ordering defers to how the
+      pairs appear on CEXs.
+
+      For cardano native tokens, ordering defers to how pairs
```

Comment:
I think I said this about token names, and noting explicitly we do not know yet how to handle conflicts. 

What if DEXs disagree on ordering. What do we do then?

Reply:
Scanning again. Yes - you've lost the bit about where native asset token names come from

Reply:
?
this is the original
```ttl
For crypto base currencies, we defer to how it appears on CEXs.
For cardano native tokens, we defer to how it appears on the main DEXs.
```
I think only order changed. But i have added back in the bit re lack of precedence